packages/api-reference/src/components/DarkModeToggle/DarkModeToggle.vue

```diff
@@ -1,18 +1,22 @@
 <script lang="ts" setup>
 import { ScalarIcon } from '@scalar/components'
 
-import { useDarkModeState } from '../../hooks/useDarkModeState'
+defineProps<{
+  isDarkMode: boolean
```

Comment:
I also learned theres a vue use package that we could use that just handles all the dark mode syncing stuff for you

https://vueuse.org/core/useDark/

Reply:
defs we can use this, only thing is it uses localStorage by default so would need something to handle the SSG part of our app ✨